swift-paperless/Views/Settings/SettingsView.swift

```diff
@@ -149,6 +149,11 @@ struct SettingsView: View {
             } label: {
                 Label(localized: .settings(.debugMenu), systemImage: "ladybug.fill")
             }
+            NavigationLink {
+                TLSListView(identityNames: Binding(get: { [] as [String] }, set: { _ in }))
```

Comment:
I'm not exactly sure what this does. Can this view actually modify the identities with this binding?

Reply:
Was a leftover from a quick fix